packages/enzyme-adapter-react-16.3/src/ReactSixteenThreeAdapter.js

```diff
@@ -133,6 +134,17 @@ function toTree(vnode) {
     case ContextProviderType: // 13
     case ContextConsumerType: // 12
       return childrenToTree(node.child);
+    case ForwardRefType: {
```

Comment:
I added this logic to the 16.3 adapter in addition to the 16 adapter because React.forwardRef was added in 16.3.